evalml/tests/component_tests/test_one_hot_encoder.py

```diff
@@ -260,6 +267,10 @@ def test_more_top_n_unique_values_large():
     encoder = OneHotEncoder(top_n=3, random_state=random_seed)
     encoder.fit(X)
     X_t = encoder.transform(X)
+
+    # Conversion changes the resulting dataframe dtype, resulting in a different random state, so we need make the conversion here too
```

Comment:
What does converting to woodwork have to do with the random state? 😨

Reply:
@freddyaboulton I believe the conversion to woodwork can update some dtypes (object --> category) which then results in differences when we sample. 😓 Check this out:
![image](https://user-images.githubusercontent.com/5422235/99438962-9015cd80-28e2-11eb-9952-692e3adaf201.png)